replit_river/session.py

```diff
@@ -267,18 +264,25 @@ async def _heartbeat(
                     ACK_BIT,
                 )
                 self._heartbeat_misses += 1
+                logging.debug(
+                    "heartbeat right now : {} {} {}".format(
```

Comment:
minor perf optimization. using `printf` style formatting should be cheaper than `String.format()` since the former won't do any formatting if the log level is INFO level or higher.

Reply:
ack